repo/fsrepo/migrations/migrations.go

```diff
@@ -0,0 +1,220 @@
+package mfsr
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+)
+
+var DistPath = "https://ipfs.io/ipns/dist.ipfs.io"
```

Comment:
let's make this overridable -- will also make it much much easier to test-drive this